nasl/CMakeLists.txt

```diff
@@ -70,14 +70,20 @@ if (NOT GPGME_FOUND)
     OUTPUT_STRIP_TRAILING_WHITESPACE)
 endif (NOT GPGME_FOUND)
 
-message (STATUS "Looking for netsnmp...")
-find_library (SNMP netsnmp)
-message (STATUS "Looking for netsnmp... ${SNMP}")
-if (SNMP)
+if (DEFINED BUILD_WITH_NETSNMP)
```

Comment:
Isn't `BUILD_WITH_NETSNMP`, due to the added block in `CmakeLists.txt`  always defined? If so, then I think you can get rid of one layer of `if` conditions :)